contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -914,6 +915,8 @@ def copy_from_changes(self, changes):
                 log_sync_exception(e, user=self.request.user, change=copy)
                 copy["errors"] = [str(e)]
                 errors.append(copy)
+                failed_copy_node = self.get_queryset().get(pk=copy["key"])
+                failed_copy_node.delete()
```

Comment:
Might want to make this defensive just in case it failed before even creating anything.